README.md

```diff
@@ -1,4 +1,77 @@
-# hubrise-webclient
-A pet JS/HTML project to see new HubRise orders in a web page
+<a href="https://www.hubrise.com/"><img src="./screenshots/hubrise-logo.png" align="left" height="48"></a>
+<br><br><br>
 
-[Demo](http://philatist-fish-74534.netlify.com)
+# HubRise {{app_name}} Template
+
+{{app_name}} is an app that prints new orders received by your HubRise location, with just a few lines of code.
+
+You can fork this repository and make changes to the code to create your own app. 
```

Comment:
The goal is more to showcase what can be achieved in 250 lines of code (HTML included!) than serving as a skeleton for a real app.